spark/src/main/resources/python/zeppelin_pyspark.py

```diff
@@ -107,6 +107,50 @@ def isAutoConvertEnabled(self):
   def isImportAllPackageUnderSparkSql(self):
     return self.version >= self.SPARK_1_3_0
 
+class PySparkCompletion:
+  def getGlobalCompletion(self):
+    objectDefList = []
+    try:
+      for completionItem in list(globals().iterkeys()):
+        objectDefList.append(completionItem)
+    except:
+      return None
+    else:
+      return objectDefList
+
+  def getMethodCompletion(self, text_value):
+    objectDefList = []
+    completion_target = text_value
+    try:
+      if len(completion_target) <= 0:
+        return None
+      if text_value[-1] == ".":
```

Comment:
I think it is possible that you want to call a private method named starting with `__` - but generally that is discouraged in python

In the following cases, autocomplete returns a list including `__add__` and other private method, so maybe it shouldn't?

![image](https://cloud.githubusercontent.com/assets/8969467/11769109/b6d4efee-a194-11e5-8d63-693b15109ead.png)

![image](https://cloud.githubusercontent.com/assets/8969467/11769111/b9d2b1ae-a194-11e5-8d6b-48aab4c04f9b.png)